src/frontend/app/features/cloud-foundry/edit-organization/edit-organization.component.scss

```diff
@@ -0,0 +1,3 @@
+.edit-org-stepper {
+  height: 100%;
```

Comment:
This seems to be c+p lots. Is it something solved by stepper-form? Does it need to go in the core stepper scss?